keras_nlp/models/bert/bert_backbone.py

```diff
@@ -28,6 +28,10 @@ def bert_kernel_initializer(stddev=0.02):
     return keras.initializers.TruncatedNormal(stddev=stddev)
 
 
+def gelu_approximate(x):
```

Comment:
There are a few other models where this is used.

https://github.com/search?q=repo%3Akeras-team%2Fkeras-nlp%20approximate%3DTrue&type=code

Maybe we should stick this in keras_utils.py, annotate it with `@keras.saving.register_keras_serializable(package="keras_nlp")`, and replace it everywhere?

Reply:
I have made this change as suggested, and also (in a seperate commit) stripped the lambda usage for all default gelu usages too. 
A search of the project for `lambda` suggests the only remaining lambda usage will not affect model saving.